client/src/components/SentimentAnalysis/SentimentAnalysis.js

```diff
@@ -149,6 +149,11 @@ const SentimentAnalysis = () => {
     setIsModalOpen(true);
   };
 
+  function toString(sentimentValue) {
```

Comment:
since `toString` is already a built in class method, perhaps a different name for clarity?

Reply:
how about `convertSentimentToString(value)`?